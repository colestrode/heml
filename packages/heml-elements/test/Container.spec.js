const render = require('@heml/render')
const Container = require('../build/Container').default
const Style = require('../build/Style').default

beforeAll(() => {
  Style.preRender({ elements: [] });
})

test('renders with some attrs', () => {
  const attrs = {
    margin: '1px 2px 3px 4px'
  }

  return expect(render.renderElement(Container, attrs, 'Lady Dynamite')).toMatchSnapshot()
});
