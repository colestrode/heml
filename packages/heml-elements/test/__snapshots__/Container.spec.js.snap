// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with some attrs 1`] = `
"<div margin=\\"1px 2px 3px 4px\\" class=\\"container\\">
  START_CONDITIONmso | IEEND_CONDITION
    <table class=\\"container__table__ie\\" role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\"><tr><td>
  END_COMMENT_CONDITIONAL
  <table class=\\"container__table\\" role=\\"presentation\\" border=\\"0\\" align=\\"center\\" cellpadding=\\"0\\" cellspacing=\\"0\\" width=\\"100%\\"><tr class=\\"container__row\\"><td class=\\"container__cell\\" width=\\"100%\\" align=\\"left\\" valign=\\"top\\">Lady Dynamite</td></tr></table>
  START_CONDITIONmso | IEEND_CONDITION
    </td></tr></table>
  END_COMMENT_CONDITIONAL
  </div>"
`;
